recipes/kmsxx/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "kmsxx" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tomba/kmsxx.git
+  git_rev: master
```

Comment:
This is not reproducible, are there no tags in the upstream project? Could it make sense to open an issue upstream to ask for them?